specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/assessments.json

```diff
@@ -0,0 +1,368 @@
+        "operationId": "Assessments_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceId"
+          },
+          {
+            "$ref": "#/parameters/AssessmentName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Assessment was deleted"
+          },
+          "204": {
+            "description": "No Content - Assessment does not exist"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "SecurityAssessmentList": {
+      "type": "object",
+      "description": "Page of a security assessments list",
+      "properties": {
+        "value": {
+          "description": "Collection of security assessments in this page",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SecurityAssessment"
+          }
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The URI to fetch the next page."
+        }
+      }
+    },
+    "SecurityAssessment": {
+      "type": "object",
+      "description": "Security assessment on a resource",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SecurityAssessmentProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/Resource"
+        }
+      ]
+    },
+    "SecurityAssessmentProperties": {
+      "type": "object",
+      "description": "Describes properties of an assessment.",
+      "properties": {
+        "resourceDetails": {
+          "$ref": "../../../common/v1/types.json#/definitions/ResourceDetails"
+        },
+        "displayName": {
+          "readOnly": true,
+          "type": "string",
+          "description": "User friendly display name of the assessment"
+        },
+        "status": {
+          "$ref": "#/definitions/AssessmentStatus"
+        },
+        "additionalData": {
+          "type": "object",
+          "description": "Additional data regarding the assessment",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "links": {
+          "$ref": "#/definitions/AssessmentLinks"
+        }
+      },
+      "required": [
+        "resourceDetails",
+        "status"
+      ]
+    },
+    "AssessmentLinks": {
+      "type": "object",
+      "description": "Links relevant to the assessment",
+      "readOnly": true,
+      "properties": {
+        "azurePortal": {
```

Comment:
I'd prefer this was "azurePortalUri" to be more clear that it is a URI.  "links" can mean a lot of things in Azure